__tests__/Unit/Components/Select/Select.test.tsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
```

Comment:
need of importing react in the test file?

Reply:
resolved